eq/server/config.h

```diff
@@ -139,6 +139,12 @@ class Config : public fabric::Config< Server, Config, Observer, Layout,
     /** @return the name of the render client executable. */
     const std::string& getRenderClient() const { return _renderClient; }
 
+    /** Set the render client command line options. */
+    void setRenderClientArgs( const Strings& args ){ _renderClientArgs = args; }
```

Comment:
Not sure yet - might also be a list of env variable patterns to transmit, e.g., 'RTNEURON_*, OSGTRANSPARENCY_*'